config/clusters/catalystproject-latam/unitefa-conicet.values.yaml

```diff
@@ -33,8 +33,6 @@ jupyterhub:
         authenticator_class: cilogon
       CILogonOAuthenticator:
         oauth_callback_url: "https://unitefa-conicet.latam.catalystproject.2i2c.cloud/hub/oauth_callback"
-        username_claim: "email"
```

Comment:
A toplevel config of username_claim is ignored when a idp specific username_claim is provided, and in oauthenticator 16, its not allowed. So, this is a helpful refactor for the z2jh upgrade to come.